projects/1.20/assets/additional-lanterns/additionallanterns/lang/zh_cn.json

```
@@ -0,0 +1,739 @@
+  "block.additionallanterns.cyan_stone_bricks_lantern": "青色石砖灯笼",
+  "block.additionallanterns.purple_stone_bricks_lantern": "紫色石砖灯笼",
+  "block.additionallanterns.blue_stone_bricks_lantern": "蓝色石砖灯笼",
+  "block.additionallanterns.brown_stone_bricks_lantern": "棕色石砖灯笼",
+  "block.additionallanterns.green_stone_bricks_lantern": "绿色石砖灯笼",
+  "block.additionallanterns.red_stone_bricks_lantern": "红色石砖灯笼",
+  "block.additionallanterns.black_stone_bricks_lantern": "黑色石砖灯笼",
+  "block.additionallanterns.stone_bricks_chain": "石砖锁链",
+  "block.additionallanterns.warped_lantern": "诡异灯笼",
+  "block.additionallanterns.white_warped_lantern": "白色诡异灯笼",
+  "block.additionallanterns.orange_warped_lantern": "橙色诡异灯笼",
+  "block.additionallanterns.magenta_warped_lantern": "品红色诡异灯笼",
+  "block.additionallanterns.light_blue_warped_lantern": "淡蓝色诡异灯笼",
+  "block.additionallanterns.yellow_warped_lantern": "黄色诡异灯笼",
+  "block.additionallanterns.lime_warped_lantern": "黄绿色诡异灯笼",
+  "block.additionallanterns.pink_warped_lantern": "粉红色诡异灯笼",
+  "block.additionallanterns.gray_warped_lantern": "灰色诡异灯笼",
+  "block.additionallanterns.light_gray_warped_lantern": "淡灰色诡异灯笼",
+  "block.additionallanterns.cyan_warped_lantern": "青色诡异灯笼",
+  "block.additionallanterns.purple_warped_lantern": "紫色诡异灯笼",
+  "block.additionallanterns.blue_warped_lantern": "蓝色诡异灯笼",
+  "block.additionallanterns.brown_warped_lantern": "棕色诡异灯笼",
+  "block.additionallanterns.green_warped_lantern": "绿色诡异灯笼",
+  "block.additionallanterns.red_warped_lantern": "红色诡异灯笼",
+  "block.additionallanterns.black_warped_lantern": "黑色诡异灯笼",
+  "block.additionallanterns.warped_chain": "诡异锁链",
+  "block.additionallanterns.waxed_copper_lantern": "涂蜡的铜灯笼",
+  "block.additionallanterns.white_waxed_copper_lantern": "白色涂蜡的铜灯笼",
+  "block.additionallanterns.orange_waxed_copper_lantern": "橙色涂蜡的铜灯笼",
+  "block.additionallanterns.magenta_waxed_copper_lantern": "品红色涂蜡的铜灯笼",
+  "block.additionallanterns.light_blue_waxed_copper_lantern": "淡蓝色涂蜡的铜灯笼",
+  "block.additionallanterns.yellow_waxed_copper_lantern": "黄色涂蜡的铜灯笼",
+  "block.additionallanterns.lime_waxed_copper_lantern": "黄绿色涂蜡的铜灯笼",
+  "block.additionallanterns.pink_waxed_copper_lantern": "粉红色涂蜡的铜灯笼",
+  "block.additionallanterns.gray_waxed_copper_lantern": "灰色涂蜡的铜灯笼",
+  "block.additionallanterns.light_gray_waxed_copper_lantern": "淡灰色涂蜡的铜灯笼",
+  "block.additionallanterns.cyan_waxed_copper_lantern": "青色涂蜡的铜灯笼",
+  "block.additionallanterns.purple_waxed_copper_lantern": "紫色涂蜡的铜灯笼",
+  "block.additionallanterns.blue_waxed_copper_lantern": "蓝色涂蜡的铜灯笼",
+  "block.additionallanterns.brown_waxed_copper_lantern": "棕色涂蜡的铜灯笼",
+  "block.additionallanterns.green_waxed_copper_lantern": "绿色涂蜡的铜灯笼",
+  "block.additionallanterns.red_waxed_copper_lantern": "红色涂蜡的铜灯笼",
+  "block.additionallanterns.black_waxed_copper_lantern": "黑色涂蜡的铜灯笼",
+  "block.additionallanterns.waxed_copper_chain": "涂蜡的铜锁链",
+  "block.additionallanterns.waxed_exposed_copper_lantern": "斑驳的涂蜡铜灯笼",
```

```suggestion
  "block.additionallanterns.waxed_exposed_copper_lantern": "涂蜡的斑驳铜灯笼",
```
1.20中采用`涂蜡的铜xx` `涂蜡的斑驳铜xx` `涂蜡的锈蚀铜xx` `涂蜡的氧化铜xx`
请注意全部修改